src/assets/headwear/towel_wrap/towel_wrap.asset.ts

```diff
@@ -0,0 +1,39 @@
+DefineAsset({
+	name: 'Towel Wrap',
+	size: 'medium',
+	graphics: 'graphics.json',
+	colorization: {
+		towel: {
+			name: 'Towel Wrap',
+			default: '#EEEEEE',
+		},
+	},
+	// size:200, y:160, centered
+	preview: 'preview.png',
+	attributes: {
+		provides: [
+			'Clothing',
+			'Headgear',
+		],
+		hides: [
+			'Hair',
+			'Ears',
+			'Fantasy_ears',
+		],
+	},
+	ownership: {
+		responsibleContributor: 'Sandrine <118102950+SandrinePDR@users.noreply.github.com>',
+		credits: ['Taja'],
+		modificationPolicy: `Fixes and New uses, otherwise ask`,
+		reusePolicy: 'Ask first',
+		licensing: [
+			{
+				part: 'towel',
+				source: 'Self-Made',
+				copyrightHolder: 'Tajo',
```

Comment:
Is this intentional or should it also be `Taja` as in credits?

Reply:
Clarified with Sandrine that it is a typo and I fixed it.

Reply:
Taja and I are fine with your fixes @ClaudiaMia